test;
[A]
    class A_testThrowableSubtype_in {
      void foo() throws Exception {
          Exception e = new Exception();
  		throw e;
      }
  }
